File: euler_032.py
from time import time

def is_pandigital(n1,n2,n3):
    return "".join(sorted(str(n1) + str(n2) + str(n3))) == "123456789"

start = time()

pandigitals = set()

for i in range(1,10000):
    j = i + 1
    while len(str(i) + str(j) + str(i * j)) < 10:
        if is_pandigital(i, j, i*j):
            pandigitals.add(i*j)
        j += 1

print("Sum of pandigital products: " + str(sum(pandigitals)))

stop = time()
print("Time: " + str(stop-start))

File: euler_032.js
const start = Date.now();

const pandigitalProduct = (x,y) => {
    const s = x.toString()
        .concat(y.toString())
        .concat((x * y).toString());
    return s.split("").sort().join("") === "123456789";
}

let pandigitalProducts = new Set();

// 1d * 4d = 4d
for (let d = 2; d < 10; d++) {
    let e = 1000;
    while (d * e < 10000) {
        if (pandigitalProduct(d,e)) {
            pandigitalProducts.add(d * e);
        }
        e++;
    }
}

// 2d * 3d = 4d
for (let d = 10; d < 100; d++) {
    let e = 100;
    while (d * e < 10000) {
        if (pandigitalProduct(d,e)) {
            pandigitalProducts.add(d * e);
        }
        e++;
    }
}

const sum = [...pandigitalProducts.values()].reduce((p,c) => p + c,0);

const end = Date.now();

console.log("Execute time: ", end - start, " Sum of pandigital products is: ", sum);